test/api-type-document.test.js

```diff
@@ -34,6 +42,54 @@ describe('<api-type-document>', function() {
     return [schema, mt];
   }
 
+  function getResponseSchema(element, model, path, methodName, statusCode) {
```

Comment:
For these functions I was usually using AmfLoader to keep it there. You can easily use it later if needed.